esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2270,6 +2272,8 @@ bool CFileSprayEx::onSprayVariable(IEspContext &context, IEspSprayVariable &req,
         if (!req.getExpireDays_isNull())
             options->setExpireDays(req.getExpireDays());
 
+        options->setNoCommon(req.getNoCommon());
```

Comment:
again, there is already code to set, see line 2264..
why was this added?

Reply:
Removed